.github/workflows/make_release.yml

```diff
@@ -3,16 +3,62 @@ name: Make release
 
 on:
   workflow_dispatch:
+    inputs:
+      dry_run:
+        description: "Dry-run"
+        type: boolean
 
 permissions:
   contents: write
 
 jobs:
   release:
+    env:
+      CARGO_REGISTRY_TOKEN: ${{ secrets.CARGO_REGISTRY_TOKEN }}
     name: Release
     runs-on: ubuntu-latest
     steps:
       - name: Checkout
         uses: actions/checkout@8e5e7e5ab8b370d6c329ec480221332ada57f0ab
         with:
           fetch-depth: 0
+
+      # This step will also publish the package to crates.io.
+      - name: Create GitHub release
+        uses: codfish/semantic-release-action@785242228c5b52cd4ada99e211ca1bb9e2bc0128
+        with:
+          branches: ${{ github.ref_name }}
+          additional_packages: |
+            [
+              '@semantic-release/github',
+              '@semantic-release/git',
+              '@semantic-release/changelog',
+              '@semantic-release/exec',
+              'semantic-release-cargo'
+            ]
+          dry_run: ${{ inputs.dry_run }}
+        env:
+          GITHUB_TOKEN: ${{ secrets.CONCRETE_ACTIONS_TOKEN }}
+
+      - name: Build web package
+        run: |
+          make build_web_js_api
+
+      - name: Publish web package
+        uses: JS-DevTools/npm-publish@0f451a94170d1699fd50710966d48fb26194d939
+        with:
+            token: ${{ secrets.NPM_TOKEN }}
+            package: tfhe/pkg/package.json
+            dry-run: ${{ inputs.dry_run }}
+
+      - name: Build Node package
+        run: |
+          make build_node_js_api
```

Comment:
Does build_node_js_api clear the directory from the previous build?

Reply:
I can remove `pkg` directory prior to that to ensure this is a fresh package.

Reply:
AFAIK, you build web package in same directory just before. That's why I suggested it.